lib/checks/navigation/region.js

```diff
@@ -62,7 +74,26 @@ function findRegionlessElms(virtualNode) {
 	}
 }
 
-var regionlessNodes = findRegionlessElms(virtualNode);
-this.relatedNodes(regionlessNodes);
+regionlessNodes = findRegionlessElms(virtualNode)
+	// Find first parent marked as having region descendant (or body) and
+	// return the node right before it as the "outer" element
+	.map(node => {
+		let vNode = axe.utils.getNodeFromTree(node);
```

Comment:
Why don't we just have `findRegionlessElms` return vitual nodes, instead of having to look them up again?